cmd/nerdctl/container_run_restart_linux_test.go

```diff
@@ -154,3 +155,28 @@ func TestUpdateRestartPolicy(t *testing.T) {
 	inspect := base.InspectContainer(tID)
 	assert.Equal(t, inspect.RestartCount, 2)
 }
+
+func TestUpdateRestartPolicyOnRunned(t *testing.T) {
```

Comment:
- In comment lines please explain how this is different from `TestUpdateRestartPolicy`
- "Runned" doesn't seem grammatically correct

Reply:
Thanks @AkihiroSuda for the PR review.

It function name has been changed , and add some comments